src/main/frontend/components/NavBar.js

```diff
@@ -8,8 +8,41 @@ import EditVenue from "./EditVenue";
 import VenueShow from "./VenueShow.js";
 
 const NavBar = (props) => {
+
+const [concertVenues, setConcertVenues] = useState([])
+const venueName = props.match.params.type
+
+const fetchConcertVenues = async () => {
+  try {
+    const response = await fetch(`/api/v1/venues/${venueId}`);
+    if (!response.ok) {
+      const errorMessage = `${response.status} (${response.statusText})`;
+      const error = new Error(errorMessage);
+      throw error;
+    }
+    const venueData = await response.json();
+    setVenue(venueData.venue);
+  } catch (error) {
+    console.error(`There was an error in fetch: ${error}`);
+  }
+}
+
+ useEffect(() => {
+   fetchConcertVenues()
+ },[venueName])
+
```

Comment:
I see the fetch request used in the `useEffect`, but I'm not clear on why this is here and what the role is.  What is the goal once you get a specific venues information?